docs/src/design/aux_table/hasher.md

```diff
@@ -238,7 +238,7 @@ When describing AIR constraints, we adopt the following notation: for column $x$
 
 ### Row address constraint
 
-As mentioned above, row address $r$ starts at $0$, and is incremented by $1$ with every row. The first condition can be enforced with a boundary constraint which specifies $r=0$ at the first row. The second condition can be enforced via the following transition constraint:
+As mentioned above, row address $r$ starts at $1$, and is incremented by $1$ with every row. The first condition can be enforced with a boundary constraint which specifies $r=1$ at the first row. The second condition can be enforced via the following transition constraint:
```

Comment:
Based on this comment, there's at least one change required earlier in the documentation as well. I searched back and found a reference at row 39 that should be updated, but there may be others as well. Can you update at row 39 and check the full document again to see if there are any other spots that need to change?

Reply:
Thanks. Fixed. Found one other instance in processor::hasher in the comments.